test/functional/api/vim_spec.lua

```diff
@@ -1591,7 +1591,10 @@ describe('API', function()
       eq({'a', '', 'b'}, meths.list_runtime_paths())
       meths.set_option('runtimepath', ',a,b')
       eq({'', 'a', 'b'}, meths.list_runtime_paths())
+      -- trailing , is ignored, use ,, if you really really want $CWD
```

Comment:
is this a change (or even existing behavior) that needs to be documented (e.g. mention it on docstring for `nvim_list_runtime_paths()`)? also not mentioned in `:h 'runtimepath'`

Reply:
It is the existing behavior for internal runtime lookup (essentially all other code paths except for nvim_list_runtime_paths()). Could have compat note perhaps in the docs but i think this was rarely used (c f at least typing `.` as a path which also works and is more explicit)